mdbook-course/src/bin/course-schedule.rs

```diff
@@ -45,7 +45,7 @@ fn timediff(actual: u64, target: u64, slop: u64) -> String {
             duration(actual),
             duration(actual - target),
         )
-    } else if actual < target - slop {
+    } else if actual + slop < target {
```

Comment:
@djmitche, are you happy with this change? I think I might have done the timing information wrong in the moved slides to trigger an integer underflow here?

Reply:
Indeed, good point!